cview

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+
+CVIEW_SERVER="$(hesinfo cview sloc)"
+
+echo "$@" | nc "$CVIEW_SERVER" 3704
```

Comment:
I am a little sad at the loss of getservbyname(), but, uh, port numbers don't change so much, and we don't appear to have an allocation from IANA (and in fact we are squatting)

adobeserver-4	3704	tcp	Adobe Server 4	[Frank_Soetebeer]	[Frank_Soetebeer]	2003-01